ScoutSuite/core/cli_parser.py

```diff
@@ -239,9 +235,32 @@ def _init_common_args_parser(self):
                             help='Name of out-of-scope services.')
         parser.add_argument('--exceptions',
                             dest='exceptions',
-                            default=None,
-                            nargs='?',
+                            default=[None],
+                            nargs='+',
                             help='Exception file to use during analysis.')
+        parser.add_argument('--result-format',
+                            dest='result_format',
+                            default='json',
+                            type=str,
+                            choices=['json', 'sqlite'],
+                            help="[EXPERIMENTAL FEATURE] The database file format to use. JSON doesn't require a server to view the report, "
+                                 "but cannot be viewed if the result file is over 400mb.")
+        parser.add_argument('--serve',
+                            dest="database_name",
+                            default=None,
+                            const=True,
+                            nargs="?",
+                            help="Serve the specified result database on the server to show the report. "
+                                 "This must be used when the results are exported as an sqlite database.")
+        parser.add_argument('--host',
+                            dest="host_ip",
+                            default="127.0.0.1",
+                            help="Address on which you want the server to listen. Defaults to localhost.")
+        parser.add_argument('--port',
+                            dest="host_port",
+                            type=int,
+                            default=8000,
+                            help="Port on which you want the server to listen. Defaults to 8000.")
```

Comment:
Can we put the server related stuff in a sub-parser or something like that?

Reply:
@zer0x64 your job (if we do that)

Reply:
This is kinda of a touchy subject, because
1. Theorically, the subparsers are meant to be providers
2. The way it is done right now, because of the execution order, you need to log in before being able to serve the report. This would require some heavier changes for now than what it's actually worth, because it touches how to provider object is created